### PR TITLE
Hybrid

### PR DESCRIPTION
**Pull request by alexanderson1993**
- Changed references of `content` to `slot`
- Changed the `paper-menu` element to be `paper-listbox`, an element with the same API that is already hybridized (no plans to hybridize `paper-menu`)
- Updated the `ready` method with Polymer 2 specific code for loading in the template
- Added code for automatically updating the scroll of the paper-listbox when the selection is changed

### CODE REVIEW

File: birch-typeahead.html [hunk 1]
```diff
@@ -335,12 +340,22 @@
...
 
       _handleUpArrow: function(e){
-        if(this._hideOptions) return;
+        if(this._hideOptions){
+          if(this.options.length){
+            e.preventDefault();
+            this._hideOptions = false;
+          }
+          return;
+        }
```
Comment (jpodwys) [inline birch-typeahead.html]: This isn't necessary as the down arrow accomplishes the same thing. Also, there is no a11y standard to support the up arrow as there is with the down arrow so it would be an unexpected behavior. I would appreciated if you would revert this change.

**Comment by alexanderson1993** [comment]
Fixes have been implemented.
